pkg/kubectl/kustomize/k8sdeps/BUILD

```diff
@@ -0,0 +1,37 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
```

Comment:
This seems like a really, really, really "novel" thing to do.  It seems like it is trying to bypass normal vendoring.

Reply:
The package `pkg/kubectl/kustomize/k8sdeps` is copied from kustomize repo. The `BUILD` file here is generated by the script `hack/godep-save.sh`. Is this `load` incorrect? How could I fix it?

Reply:
This load is correct. I suspect Clayton is talking about the package being copied in.